docker/sf-gpu.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM tensorflow/tensorflow:2.10.1-gpu
+RUN pip install -U secretflow -i https://pypi.doubanio.com/simple/
+RUN pip install jupyter -i https://pypi.doubanio.com/simple/
```

Comment:
`jupyter` is not a required package of SecretFlow, could you remove jupyter in dockerfile? It's a violation of the principle of minimization.
You can give a suggestion about how to install jupyter for user in somewhere, e.g. in installation guide.